Atomic/diff.py

```diff
@@ -8,6 +8,12 @@
 from . import Atomic
 
 class Diff(Atomic):
+
+    def diff_tty(self):
```

Comment:
So would you then recommend that I have a member variable for Atomic called is_dbus which is default set to False?